src/EnergyPlus/PurchasedAirManager.cc

```diff
@@ -2245,6 +2245,15 @@ namespace PurchasedAirManager {
                     }
                 }
 
+                // If cooling capacity is limited to zero, SupplyMassFlowRate* should be set to zero
+                if (((PurchAir(PurchAirNum).CoolingLimit == LimitCapacity) ||
+                    (PurchAir(PurchAirNum).CoolingLimit == LimitFlowRateAndCapacity)) &&
+                    (PurchAir(PurchAirNum).MaxCoolTotCap == 0)) {
+                    SupplyMassFlowRateForCool = 0;
+                    SupplyMassFlowRateForDehum = 0;
+                    SupplyMassFlowRateForHumid = 0;
```

Comment:
Isn't humidification a heating process? I don't understand why its flow rate is zeroed out.

I imagine the same issue exists for heating limits.

Reply:
This is the cooling section, so we want everything off here unless there's OA flow. Likewise, the dehumid mass flow is set to zero in the heating section.